geoh5py/objects/surveys/electromagnetics/base.py

```diff
@@ -34,35 +34,34 @@
 from geoh5py.data import FloatData, IntegerData, ReferencedData
 from geoh5py.groups.property_group import PropertyGroup
 from geoh5py.objects import Curve
-from geoh5py.objects.object_base import ObjectBase
+from geoh5py.objects.surveys.base import BaseSurvey
 from geoh5py.shared.utils import str2uuid, str_json_to_dict
 
 
 if TYPE_CHECKING:
     from geoh5py.groups import Group
     from geoh5py.workspace import Workspace
 
-TYPE_MAP = {
-    "Transmitters": "transmitters",
-    "Receivers": "receivers",
-    "Base stations": "base_stations",
-}
-OMIT_LIST = [
+OMIT_LIST = (
```

Comment:
Make them calss attributes if they are associated to the class

Reply:
Can try